sensor_msgs/CompressedImage compressedImage 
float32[16] flat_transform
bool is_stable
sensor_msgs/CameraInfo cameraInfo
